src/renderers/shared/fiber/ReactFiberExpirationTime.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule ReactFiberExpirationTime
+ * @flow
+ */
+
+'use strict';
+
+import type {PriorityLevel} from 'ReactPriorityLevel';
+const {
+  NoWork,
+  SynchronousPriority,
+  TaskPriority,
+  HighPriority,
+  LowPriority,
+  OffscreenPriority,
+} = require('ReactPriorityLevel');
+
+const invariant = require('fbjs/lib/invariant');
+
+// TODO: Use an opaque type once ESLint et al support the syntax
+export type ExpirationTime = number;
+
+const Done = 0;
+const Sync = 1;
+const Task = 2;
+const Never = Number.MAX_SAFE_INTEGER;
+
+const UNIT_SIZE = 10;
+const MAGIC_NUMBER_OFFSET = 10;
+
+exports.Done = Done;
```

Comment:
How about keeping the `NoWork` name here? I also wonder if the value should be named something that has something that indicates that it is really a dirty flag and that this essentially means false. `Done` stood out to me as not indicating that well unless you already know how it works.

Reply:
Yeah I'm not happy with any of the names here, was just trying to avoid a naming clash. `NoWork` seems fine, can import the other one as `NoWorkPriority` if I need to.